reservations/models.py

```diff
@@ -16,6 +18,36 @@
 User = get_user_model()
 
 
+RESERVATION_FORM_FIELDS = (
```

Comment:
I see duplicate list is used in migration `0019_reservationmetadata.py`. 
I did not find any usage of this from this PR. Where is this list used?

Reply:
Good catch. This was originally used in the admin model for validating the `form_field` value. But I don't think that validation is needed because the user cannot add/modify/delete values metadata fields in Django admin, so I removed it. This `RESERVATION_FORM_FIELDS` can actually be removed now.